exercises/word-search/package.json

```diff
@@ -11,7 +11,7 @@
   "devDependencies": {
     "@types/jest": "^24.0.18",
     "@types/node": "^12.7.12",
-    "@typescript-eslint/eslint-plugin": "^2.5.0",
+    "@typescript-eslint/eslint-plugin": "^2.3.3",
```

Comment:
Why this?

Reply:
The make command complained that the word-search package.json was not matching the package.json that is in the root/common dir. 2.3.3 is what was in the common package.json.

Reply:
I'm more wondering why this was not 2.3.3 in the first place :/.

Reply:
Ah I figured it out. The CI was broken when this exercise was added, so we didn't actually check this. Thanks for fixing!